src/QMCWaveFunctions/Fermion/MultiSlaterDeterminantFast.cpp

```diff
@@ -300,7 +306,7 @@ WaveFunctionComponent::GradType MultiSlaterDeterminantFast::evalGrad(ParticleSet
 {
   if (usingBF)
   {
-    APP_ABORT("Fast MSD+BF: evalGrad not implemented. \n");
+    throw std::runtime_error("Fast MSD+BF: evalGrad not implemented. \n");
```

Comment:
could you please add function BackFlowStopper(const std::string& func_name) which replace this kind of if statement in all places.

Reply:
Done